packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -472,14 +473,16 @@ class DropdownButton<T> extends StatefulWidget {
     @required this.items,
     this.value,
     this.hint,
+    this.disabledHint,
     @required this.onChanged,
     this.elevation = 8,
     this.style,
     this.iconSize = 24.0,
     this.isDense = false,
-  }) : assert(items != null),
-       assert(value == null || items.where((DropdownMenuItem<T> item) => item.value == value).length == 1),
-      super(key: key);
+  })  : assert(items == null
+      || value == null
+      || items.where((DropdownMenuItem<T> item) => item.value == value).length == 1),
```

Comment:
the formatting on this assert is a bit confusing. Consider something like:

```dart
  })  : assert(items == null || value == null ||
               items.where((DropdownMenuItem<T> item) => item.value == value).length == 1),
```

Reply:
done